manticore/__main__.py

```diff
@@ -63,13 +63,111 @@ def parse_arguments():
 
     return parsed
 
+
+def ethereum_cli(args):
+    from seth import ManticoreEVM
+
+    m = ManticoreEVM(procs=args.procs)
+
+    with open(args.argv[0]) as f:
+        source_code = f.read()
+
+    user_account = m.create_account(balance=1000)
+    contract_account = m.solidity_create_contract(source_code, owner=user_account)
+    attacker_account = m.create_account(balance=1000)
+
+    last_coverage = None
+    new_coverage = 0
+    tx_count = 0
+    while new_coverage != last_coverage and new_coverage < 100:
+
+        symbolic_data = m.make_symbolic_buffer(320)
+        symbolic_value = m.make_symbolic_value()
+
+        m.transaction(caller=attacker_account,
+                         address=contract_account,
+                         data=symbolic_data,
+                         value=symbolic_value )
+
+        tx_count += 1
+        last_coverage = new_coverage
+        new_coverage = m.global_coverage(contract_account)
+
+        print "[+] Coverage after %d transactions: %d%%"%(tx_count, new_coverage)
+        print "[+] There are %d reverted states now"% len(m.terminated_state_ids)
+        print "[+] There are %d alive states now"% len(m.running_state_ids)
+
+    for state in m.all_states:
+        print str(state.context['last_exception'])
+
+    # for state in seth.all_states:
```

Comment:
Can move this to seth.report() or to set.generate_testcase() ?  Or is that a diff PR ?

Reply:
We are going to ultimately delete this code and implement this design with the workspace oriented output: https://github.com/orgs/trailofbits/teams/manticore/discussions/3

I just wanted to keep the code around for reference, for now.

Reply:
Okay a different PR it is!